EIPS/eip-1052.md

```diff
@@ -24,7 +24,7 @@ takes one argument from the stack, zeros the first 96 bits
 and pushes to the stack the keccak256 hash of the code of the account 
 at the address being the remaining 160 bits. 
 
-In case the account does not exist `0` is pushed to the stack.
+In case the account does not exist or is empty `0` is pushed to the stack.
```

Comment:
```suggestion
In case the account does not exist or is empty (as defined by [EIP-161](https://eips.ethereum.org/EIPS/eip-161)) `0` is pushed to the stack.
```

Reply:
@chfast can you approve?

Reply:
:+1:

Reply:
![Screenshot 2019-11-22 at 22 13 25](https://user-images.githubusercontent.com/20340/69464092-5369d180-0d75-11ea-9df7-6b33bd438bf4.png)

It seems @sorpaas has disabled maintainers' access to his fork. Will need to create a new PR.

Reply:
Done in #2388.